cloudformation/fsx-substack.cfn.json

```diff
@@ -605,10 +605,18 @@
             ]
           },
           "DataCompressionType": {
-            "Fn::Select": [
-              "19",
+            "Fn::If": [
+              "UseBackupId",
               {
-                "Ref": "FSXOptions"
+                "Ref": "AWS::NoValue"
+              },
+              {
+                "Fn::Select": [
+                  "19",
+                  {
+                    "Ref": "FSXOptions"
```

Comment:
What about if we don't specify `DataCompressionType`? Are we going to use "NONE" as DataCompressionType?

Reply:
As we discussed offline, the FSx default of DataCompressionType is NONE, so our logic is inline with that. We cannot use `AWS:NoValue` as the default because of an issue with update, and we did the change in this commit: https://github.com/aws/aws-parallelcluster/commit/6c04a0e41ba25c3bc5d0df99c97f5d97713ac4e1